build.gradle

```diff
@@ -437,6 +439,7 @@ project('scratchpad') {
         compile project(':main')
         compile "commons-codec:commons-codec:${commonsCodecVersion}"
         compile "org.apache.commons:commons-math3:${commonsMathVersion}"
+        implementation "org.apache.logging.log4j:log4j-api:${log4jVersion}"
```

Comment:
can you stick with 'compile' and we can change all compile to implementation later